samples/GalleryApp/Pages/ObjectDetectorPage.xaml

```diff
@@ -106,7 +116,13 @@
                         <TextBox Name="UISkillOutputDescription" IsReadOnly="True" AcceptsReturn="True"/>
                         <TextBlock Text="Available execution devices:" FontWeight="Bold"/>
                         <ListBox Name="UISkillExecutionDevices" SelectionChanged="UISkillExecutionDevices_SelectionChanged"/>
-                        <ListView Name="UIObjectKindFilters" SelectionMode="Multiple" ScrollViewer.VerticalScrollBarVisibility="Visible" SelectionChanged="UIObjectKindFilters_SelectionChanged" />
+                        <StackPanel Orientation="Horizontal">
+                            <CheckBox x:Name="TriStateCheckBox" Content="Select All" IsThreeState="True"
```

Comment:
>TriStateCheckBox [](start = 46, length = 16)

nit: uniformize UI variable names throughout, i.e. I've been using the prefix "UI" for all controls declared in xaml